indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -286,10 +340,17 @@
             self.$categoryResults.html('');
             _.each(categories, function(category) {
                 var $result = self._buildSubcategory(category, true);
-                $result.find('.icon-wrapper').append($('<i>', {
-                    class: 'icon-search',
-                    title: $T.gettext("Search result")
-                }));
+                if (category.is_favorite) {
+                    $result.find('.icon-wrapper').append($('<i>', {
+                        class: 'icon-star',
```

Comment:
The icon is not properly aligned on the blue background.
And IMHO it looks a bit too "important". Almost like a button..